client/src/api/schema/schema.ts

```diff
@@ -8957,6 +8971,21 @@ export interface components {
              */
             time: string;
         };
+        /** MetricsSummaryCumulative */
+        MetricsSummaryCumulative: {
+            /** Total Allocated Cores Cpu */
+            total_allocated_cores_cpu: number;
+            /** Total Allocated Memory Mebibyte */
+            total_allocated_memory_mebibyte: number;
+            /** Total Energy Needed Cpu Kwh */
+            total_energy_needed_cpu_kwh: number;
+            /** Total Energy Needed Kwh */
+            total_energy_needed_kwh: number;
+            /** Total Energy Needed Memory Kwh */
+            total_energy_needed_memory_kwh: number;
```

Comment:
Can you move that into another model ?

Reply:
Do you have something like this in mind?:

```python
# lib/galaxy/schema/schema.py

class MetricsSummaryEnergyCumulative(Model):
    total_energy_needed_cpu_kwh: float
    total_energy_needed_memory_kwh: float
    total_energy_needed_kwh: float

class MetricsSummaryCoreCumulative(Model):
    total_allocated_cores_cpu: int
    total_allocated_memory_mebibyte: int
    total_runtime_seconds: int
    energy_usage_metrics: MetricsSummaryEnergyCumulative
```